### DISCUSSION TITLE
Updating parent data from child form like ->editOptionForm

### DISCUSSION THREAD

**Discussion by Nestoran** [filamentphp/filament#6517] [Help] [answered]
Is there a way to modify/edit the data of an existing parent record from the form of a CreateChild/EditChild page?
Similar to the ```->createOptionForm()``` , but instead opening an empty modal for creating a new parent record, something like an ```->createOrEditOptionForm()```.
This would create the same form but if an option was already selected, then the form would be filled with the selected records data, and upon sending the form, the parent record's data would be updated.

Instead of this:

```php
Forms\Components\Select::make('parent_id')
    ->relationship('parent', 'name')
    ->createOptionForm([
            Forms\Components\TextInput::make('name')
                ->required(),
            Forms\Components\TextInput::make('description')
                ->required(),
    ])
```

Something like this, allowing to edit/update selected parent record:

```php
Forms\Components\Select::make('parent_id')
    ->relationship('parent', 'name')
    ->createOrEditOptionForm([
            Forms\Components\TextInput::make('name')
                ->required(),
            Forms\Components\TextInput::make('description')
                ->required(),
    ])
```

Or can this be achieved in any way, without navigating to the parent resource's edit page?

**Comment by danharrin** [comment, marked as answer]
This has been implemented in v3 👍 But in v2, you can't have more than one action next to a field
